persist/storer.go

```diff
@@ -56,8 +54,12 @@ type inMemoryStore struct {
 
 // Holder for any entry in the JSON storage
 type jsonEntry struct {
+	entryTimestamp
```

Comment:
If we are going to `json.Marshal` this, I think we need to
```suggestion
	entryTimestamp `json:",inline"`
```

Otherwise the go library will create an `entryTimestamp` node.

Reply:
inline is the default behavior for embedded structs i think but i will add it to be more explicit , thanks!